R/get_gambl_colours.R

```diff
@@ -0,0 +1,288 @@
+#' @title Get GAMBL Colours.
```

Comment:
Does this function really belong to this repo? I probably would have sorted it into the utility bin. That being said, I can understand the rationale for having it here as well. Just wanted to see if this is something that was discussed at some point.

Reply:
It can also be in the results child because it is a `get_` function, but I agree with it being here in this repo - because this would be a "one-stop-shop" for all the visuals, where the standard colors are at the core. We can also always find a better home for it if this particular repo will be not efficient

Reply:
Exactly, I just realized that many plotting functions are calling this function. So if it were to be moved, the function would have to be prefixed with the correct repo. So. it makes sense to leave it here, for sure.